charts/opensearch/values.yaml

```diff
@@ -343,173 +128,763 @@ securityConfig:
+  # To add secrets to the keystore:
+  #  - secretName: opensearch-encryption-key
+
+  # Deprecated
+  # please use the above podSecurityContext.fsGroup instead
+  fsGroup: ""
+
+  ## Set optimal sysctl's through securityContext. This requires privilege. Can be disabled if
+  ## the system has already been preconfigured. (Ex: https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
+  ## Also see: https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/
+  sysctl:
     enabled: false
 
-# Deprecated
-# please use the above podSecurityContext.fsGroup instead
-fsGroup: ""
-
-## Set optimal sysctl's through securityContext. This requires privilege. Can be disabled if
-## the system has already been preconfigured. (Ex: https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
-## Also see: https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/
-sysctl:
-  enabled: false
-
-## Set optimal sysctl's through privileged initContainer.
-sysctlInit:
-  enabled: false
+  ## Set optimal sysctl's through privileged initContainer.
+  sysctlInit:
+    enabled: false
   # override image, which is busybox by default
   # image: busybox
   # override image tag, which is latest by default
   # imageTag:
 
-## Enable to add 3rd Party / Custom plugins not offered in the default OpenSearch image.
-plugins:
-  enabled: false
-  installList: []
+  ## Enable to add 3rd Party / Custom plugins not offered in the default OpenSearch image.
+  plugins:
+    enabled: false
+    installList: []
   # - example-fake-plugin
 
-# -- Array of extra K8s manifests to deploy
-extraObjects: []
-  # - apiVersion: secrets-store.csi.x-k8s.io/v1
-  #   kind: SecretProviderClass
-  #   metadata:
-  #     name: argocd-secrets-store
-  #   spec:
-  #     provider: aws
-  #     parameters:
-  #       objects: |
-  #         - objectName: "argocd"
-  #           objectType: "secretsmanager"
-  #           jmesPath:
-  #               - path: "client_id"
-  #                 objectAlias: "client_id"
-  #               - path: "client_secret"
-  #                 objectAlias: "client_secret"
-  #     secretObjects:
-  #     - data:
-  #       - key: client_id
-  #         objectName: client_id
-  #       - key: client_secret
-  #         objectName: client_secret
-  #       secretName: argocd-secrets-store
-  #       type: Opaque
-  #       labels:
-  #         app.kubernetes.io/part-of: argocd
-  # - |
-  #    apiVersion: policy/v1
-  #    kind: PodDisruptionBudget
-  #    metadata:
-  #      name: {{ template "opensearch.uname" . }}
-  #      labels:
-  #        {{- include "opensearch.labels" . | nindent 4 }}
-  #    spec:
-  #      minAvailable: 1
-  #      selector:
-  #        matchLabels:
-  #          {{- include "opensearch.selectorLabels" . | nindent 6 }}
+data:
+  name: "opensearch-cluster-data"
+  clusterName: "opensearch-cluster"
+  nodeGroup: "data"
+  serviceName: "opensearch-cluster-data"
```

Comment:
setting data node service name to `opensearch-cluster-data` will be different with kse 3.5's OpenSearch

Reply:
Here is the same as 3.5, is configured two serverice, one is master-service and one is data-service.